src/bundle/Resources/public/scss/_dashboard.scss

```diff
@@ -1,6 +1,14 @@
 .ez-dashboard__header {
     display: grid;
-    grid-template-columns: auto 6.25rem;
+    grid-template-columns: auto 6.5rem;
+
+    .ez-dashboard__create {
+    	.ez-btn--cotf-create {
+    		.ez-icon {
```

Comment:
Please, correct indentation to 4 spaces instead of 8. It should be applied here and the line below.